libs/auth/testing/src/drivers/in-memory/login/login.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import {
+  DaffLoginServiceInterface,
+  DaffLoginRequest,
+  DaffLoginResponse
+} from '@daffodil/auth';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffInMemoryLoginService implements DaffLoginServiceInterface<DaffLoginRequest, DaffLoginResponse> {
+  url = '/api/auth/';
+
+  constructor(private http: HttpClient) {}
+
+  login({email, password}: DaffLoginRequest): Observable<DaffLoginResponse> {
+    return this.http.post<DaffLoginResponse>(`${this.url}login`, {email, password});
+  }
```

Comment:
can't this just be: 
```
login(request: DaffLoginRequest): Observable<DaffLoginResponse> {
    return this.http.post<DaffLoginResponse>(`${this.url}login`, request);
}
```
?

Reply:
ah yeah it can. I think I was doing something else with `email` and `password` initially that I later removed.